src/main/java/org/opensearch/ad/model/AnomalyResult.java

```diff
@@ -63,13 +66,12 @@
     public static final String USER_FIELD = "user";
     public static final String TASK_ID_FIELD = "task_id";
     public static final String MODEL_ID_FIELD = "model_id";
-    public static final String TOTAL_UPDATES_FIELD = "totalUpdates";
     public static final String START_OF_ANOMALY_FIELD = "startOfAnomaly";
     public static final String IN_HIGH_SCORE_REGION_FIELD = "inHighScoreRegion";
-    public static final String RELATIVE_INDEX_FIELD = "relativeIndex";
-    public static final String CURRENT_TIME_ATTRIBUTION_FIELD = "currentTimeAttribution";
-    public static final String OLD_VALUES_FIELD = "oldValues";
-    public static final String EXPECTED_VAL_LIST_FIELD = "expectedValuesList";
+    public static final String APPROX_ANOMALY_START_FIELD = "approx_anomaly_start_time";
+    public static final String RELEVANT_ATTRIBUTION_FIELD = "relavantAttribution";
+    public static final String PAST_VALUE_FIELD = "past_value";
+    public static final String EXPECTED_VALUE_FIELD = "expected_value";
```

Comment:
since these will be arrays, how about make them plural? `past_values` / `expected_values`?

Reply:
yes, changed